### PR TITLE
[FLINK-26659] Document UI access via Ingress

### PR DESCRIPTION
**Pull request by morhidi**

### CODE REVIEW

**Comment by morhidi** [comment]
cc @wangyang0918

File: docs/content/docs/operations/ingress.md [hunk 1]
```diff
@@ -24,4 +24,36 @@ specific language governing permissions and limitations
...
+  namespace: default
+  name: basic-ingress
+spec:
+  image: flink:1.14.3
+  flinkVersion: v1_14
+  ingressDomain: flink.k8s.io
+  ...
+```
+
+The Operator takes the `ingressDomain`, `name` and `namespace` values from the deployment and creates an Ingress using the template `{{name}}.{{namespace}}.{{ingressDomain}}` to generate the host. This requires that anything `*.flink.k8s.io` should be routed to the Ingress controller on the Kubernetes cluster.
```
Comment (mbalassi) [inline docs/content/docs/operations/ingress.md:47]: Please add the following link to `Ingress` in this sentence:

https://kubernetes.io/docs/concepts/services-networking/ingress/

File: docs/content/docs/operations/ingress.md [hunk 2]
```diff
@@ -24,4 +24,36 @@ specific language governing permissions and limitations
...
+
+```shell
+kubectl get ingress
+NAME             CLASS   HOSTS                                 ADDRESS        PORTS   AGE
+basic-ingress    nginx   basic-ingress.default.flink.k8s.io    192.168.49.2   80      30m
+basic-ingress2   nginx   basic-ingress2.default.flink.k8s.io   192.168.49.2   80      3m39s
+
```
Comment (mbalassi) [inline docs/content/docs/operations/ingress.md:54]: Remove 2 empty lines from code snippet

**Review by mbalassi — CHANGES_REQUESTED** [review]
Thanks, please address 2 minor comments.

File: docs/content/docs/operations/ingress.md (hunk 1 above)
Comment (morhidi) [reply docs/content/docs/operations/ingress.md]: Added, thanks @mbalassi

File: docs/content/docs/operations/ingress.md (hunk 2 above)
Comment (morhidi) [reply docs/content/docs/operations/ingress.md]: Removed, thanks

**Review by wangyang0918 — APPROVED** [review]
Thanks for adding this useful doc. I am giving the pass now.

**Review by mbalassi — APPROVED** [review]
LGTM, merging since @wangyang0918 also approved.

**Pull request merged**